### PR TITLE
Bug 1497507 - aligning cta w. value proposition

### PR DESCRIPTION
**Pull request by gvn**
I can't assign a reviewer, so @k88hudson or @ScottDowne R?

### CODE REVIEW

**Comment by k88hudson** [comment]
Taskcluster failures are unrelated, disregard

File: content-src/components/PocketLoggedInCta/_PocketLoggedInCta.scss [hunk 1]
```diff
@@ -27,6 +27,7 @@
   .cta-text {
     font-weight: normal;
     font-size: 13px;
+    line-height: 1.230769231; // (16 / 13) – 16px computed
```
Comment (ScottDowne) [inline content-src/components/PocketLoggedInCta/_PocketLoggedInCta.scss]: "more recommendations" is no longer aligned: 
align
I think it just needs a matching line-height at the least.

File: content-src/components/Sections/_Sections.scss [hunk 2]
```diff
@@ -84,11 +84,6 @@
       font-size: 13px;
     }
   }
-
-  @media (min-width: $break-point-large) {
-    line-height: 16px;
-    height: 16px;
```
Comment (ScottDowne) [inline content-src/components/Sections/_Sections.scss]: This does change `top-stories-bottom-container`'s height to 20.8px from 16px; is that intended?

Comment (gvn) [reply content-src/components/Sections/_Sections.scss]: Well, we really shouldn't be setting fixed heights on anything containing live text unless there's a good reason to, so...yes?

Comment (gvn) [reply content-src/components/Sections/_Sections.scss]: Why was height set in the first place?

Comment (ScottDowne) [reply content-src/components/Sections/_Sections.scss]: I *think* it was something to do with the height transition for collapsing the panel, it was causing some jitter and movement for the highlights container. Something I currently cannot reproduce.

Comment (ScottDowne) [reply content-src/components/Sections/_Sections.scss]: Other than that, I cannot really see what it's doing.

Comment (gvn) [reply content-src/components/Sections/_Sections.scss]: Ok, I think it's probably safe to remove then.

**Comment by gvn** [comment]
@ScottDowne - Made that line height fixup...I think it's good now?

**Pull request merged**